debian/build.go

```diff
@@ -291,6 +292,25 @@ func getKubeadmKubeletConfigFile(v version) (string, error) {
 	}
 }
 
+// CNI get bumped in 1.9, which is incompatible for kubelet<1.9.
+// So we need to restrict the CNI version when install kubelet.
+func getKubeletCNIVersion(v version) (string, error) {
+	sv, err := semver.Make(v.Version)
+	if err != nil {
+		return "", err
+	}
+
+	v190, err := semver.Make("1.9.0")
+	if err != nil {
+		return "", err
+	}
+
+	if sv.GTE(v190) {
+		return fmt.Sprintf(">= %s", cniVersion), nil
+	}
+	return fmt.Sprint("<= 0.5.1"), nil
```

Comment:
it's a little weird to have this constant floating here, but otherwise this looks fine to me.